scripts/thyra-installer.py

```diff
@@ -0,0 +1,132 @@
+import urllib.request
+import subprocess
+import shutil
+import os
+import zipfile
+import ctypes
+from urllib.error import URLError
+from tarfile import ReadError
+
+# This file is to be bundled with pyinstaller in order to produce a .exe that can run on Windows without Python installed.
+
+THYRA_URL = "https://github.com/massalabs/thyra/releases/latest/download/thyra-server_windows_amd64"
+THYRA_FILENAME = "thyra-server.exe"
+THYRA_CONFIG_FOLDER_PATH = os.path.expanduser("~") + "\\.config\\thyra"
+ACRYLIC_DNS_PROXY_URL = "https://sourceforge.net/projects/acrylic/files/Acrylic/2.1.1/Acrylic-Portable.zip/download"
+ACRYLIC_DNS_PROXY_FILENAME = "Acrylic-Portable.zip"
+ACRYLIC_HOST_FILE = "AcrylicHosts.txt"
+DEFAULT_ACRYLIC_PATH = "C:\Program Files (x86)\Acrylic DNS Proxy"
+MKCERT_URL = "https://dl.filippo.io/mkcert/latest?for=windows/amd64"
+MKCERT_FILENAME = "mkcert.exe"
+CERTIFICATIONS_FOLDER = THYRA_CONFIG_FOLDER_PATH + "\\certs"
+
+def downloadFile(url, filename):
+    print("Downloading " + filename + "...")
+    try:
+        with urllib.request.urlopen(url) as response, open(filename, 'wb') as out_file:
+            shutil.copyfileobj(response, out_file)
+    except URLError as err:
+        print("Failed to download " + filename + " :")
+        printErrorAndExit(err)
+    print(filename + " downloaded successfully")
+
+def unzipAcrylic():
+    try: 
+        os.mkdir(DEFAULT_ACRYLIC_PATH)
+        shutil.move(ACRYLIC_DNS_PROXY_FILENAME, DEFAULT_ACRYLIC_PATH)
+        os.chdir(DEFAULT_ACRYLIC_PATH)
+    except OSError as err: 
+        printErrorAndExit(err)
+    try:
+        with zipfile.ZipFile(DEFAULT_ACRYLIC_PATH + "\\" +  ACRYLIC_DNS_PROXY_FILENAME, 'r') as zip_ref:
+            zip_ref.extractall(DEFAULT_ACRYLIC_PATH)
+        print("Acrylic unzipped")
+    except ReadError as err:
+        printErrorAndExit(err)
+
+def executeOSCommandOrFile(command, decodeBinary, errorChecking=True):
+    process = subprocess.Popen(command,
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE,
+                            universal_newlines=decodeBinary)
+    stdout, stderr = process.communicate()
+
+    if stderr != None and stderr != "" and errorChecking == True:
+        print("Error encountered while executing : " + str(command) + " :\n", stderr)
+        os._exit(-1)
```

Comment:
Why not using printErrorAndExit here like before?

Reply:
No reason I forgot, will update